documentation/models.md

```diff
@@ -0,0 +1,31 @@
+# Models
+
+In order to interact with the SQL database, why use `xo` which is a command-line tool used to generate Go code based on a database schema.
```

Comment:
```why use `xo` which is a command-line tool``` is "**why**" intended ?

Reply:
It seems it probably wasn't, should be fixed now.